Remotion/Validation/UnitTests/Attributes/Validation/LengthValidationAttributeTest.cs

```diff
@@ -29,33 +29,48 @@ namespace Remotion.Validation.UnitTests.Attributes.Validation
   [TestFixture]
   public class LengthValidationAttributeTest
   {
-    private LengthValidationAttribute _attribute;
+    private LengthValidationAttribute _lengthValidationAttribute;
+    private LengthValidationAttribute _exactLengthValidationAttribute;
+    private LengthValidationAttribute _minimumLengthValidationAttribute;
+    private LengthValidationAttribute _maximumLengthValidationAttribute;
     private Mock<IValidationMessageFactory> _validationMessageFactoryStub;
 
     [SetUp]
     public void SetUp ()
     {
-      _attribute = new LengthValidationAttribute(10, 20);
+      _lengthValidationAttribute = new LengthValidationAttribute() { MinLength = 10, MaxLength = 20 };
+      _exactLengthValidationAttribute = new LengthValidationAttribute() { MinLength = 10, MaxLength = 10 };
+      _minimumLengthValidationAttribute = new LengthValidationAttribute() { MinLength = 10 };
+      _maximumLengthValidationAttribute = new LengthValidationAttribute() { MaxLength = 10 };
       _validationMessageFactoryStub = new Mock<IValidationMessageFactory>();
     }
 
     [Test]
     public void Initialization ()
     {
-      Assert.That(_attribute.MinLength, Is.EqualTo(10));
-      Assert.That(_attribute.MaxLength, Is.EqualTo(20));
+      Assert.That(_lengthValidationAttribute.MinLength, Is.EqualTo(10));
+      Assert.That(_lengthValidationAttribute.MaxLength, Is.EqualTo(20));
+
+      Assert.That(_exactLengthValidationAttribute.MinLength, Is.EqualTo(10));
+      Assert.That(_exactLengthValidationAttribute.MaxLength, Is.EqualTo(10));
+
+      Assert.That(_minimumLengthValidationAttribute.MinLength, Is.EqualTo(10));
+      Assert.That(_minimumLengthValidationAttribute.MaxLength, Is.EqualTo(0));
+
+      Assert.That(_maximumLengthValidationAttribute.MinLength, Is.EqualTo(0));
+      Assert.That(_maximumLengthValidationAttribute.MaxLength, Is.EqualTo(10));
```

Comment:
Here we're testing if the properties work. since we're excercising the properties anyway and they have been implemented as autoproperties, there is no benefit. originally, we tested if the properties returned the values from the ctor and where assigned in the correct "order" (min to min, max to max). 

What you could do is write two tests: GetAndSetMinimumLength and GetAndSetMaximumLength. There, you first assert if the property is zero, then assign it a value and then assert that it has the new value.